getting_started/intel_debug/suggestions.rst

```diff
@@ -294,10 +294,31 @@ driver:
    #define SOC_ES8336_HEADSET_MIC1		BIT(8)
 
 
-The default quirk value for the platform can be read from
+The default and actual quirk values for the platform can be got
+from the dmesg log file (/var/log/dmesg) in the line like:
```

Comment:
There is `/var/log/dmesg` file on Fedora or Arch Linux, it's distro-specific.

Don't recommend the `dmesg` command either because it's a ring buffer that can run out.

There are countless pages on the Internet that explain what each distro does with kernel logs, just let users find that themselves:
```suggestion
in the kernel logs in a line that looks like:
```

Reply:
fixed